CoEpi/native/NativeCore.swift

```diff
@@ -43,10 +43,48 @@ extension NativeCore: ServicesBootstrapper {
     // WARN: Must not run long operations. Currently executed synchronously during startup.
     func bootstrap(dbPath: String) -> Result<(), ServicesError> {
         let libResult: LibResult<ArbitraryType>? = bootstrap_core(dbPath)?.toLibResult()
+        NSLog("%@", "blah");
+        let x = register_callback{ (myInt, myBool, myCFString) in
+            if let cfStr: CFString = myCFString {
+                logToiOS(string: cfStr as String)
+            }
+        }
+        NSLog("register_callback returned : %d", x );
         return libResult?.toVoidResult().mapErrorToServicesError() ?? libraryFailure()
     }
 }
 
+func logToiOS(string: String){
+    let cs = ConsoleLog.init()
+    cs.d(string, tags: [.db])
+}
+
+func logToiOS(logMessage: CoreLogMessage){
```

Comment:
Why do we have 2 versions? Are we going to use plain strings or the struct?

Reply:
Struct. String were used in first iteration.

Reply:
Ok, can you please remove the outdated stuff?

Reply:
Removed.